bgpd/bgp_rpki.c

```diff
@@ -649,107 +610,82 @@ static void bgpd_sync_callback(struct event *thread)
 			rrp->prefix = prefix;
 			rrp->afi = afi;
 			rrp->safi = safi;
-			event_add_event(bm->master, rpki_revalidate_prefix, rrp,
-					0, &bgp->t_revalidate[afi][safi]);
+			event_add_event(bm->master, rpki_revalidate_prefix, rrp, 0,
```

Comment:
whitespace change?

Reply:
Most likely got reformatted somehow, I'll return it back.